tests/factories/attributes.py

```diff
@@ -9,7 +11,7 @@
 USER_ID = Faker("hexify", text="^" * 40)
 H_USERNAME = Faker("hexify", text="^" * 30)
 H_DISPLAY_NAME = Faker("name")
-H_USERID = Sequence(lambda n: f"acct:user_{n + 1}@example.com")
```

Comment:
My theory with concurrent executions of the tests sometimes causes collisions with this.